recipes/r-rgenoud/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+export DISABLE_AUTOBREW=1
+${R} CMD INSTALL --install-tests --build . ${R_ARGS}
```

Comment:
This includes the tests into the to be uploaded package. Do we want that?

Does testing also work by adding:
```
source_files:
  - tests
```

to the test section (and removing the `--install-tests` arg)?

Reply:
I've done it that way before, but it makes the testing command a bit more verbose ([example](https://github.com/conda-forge/r-vetiver-feedstock/blob/348dc67d79cd64b7a6670270e5c0e3a304ad3269/recipe/meta.yaml#LL89C57-L89C57)). Additionally, I've had issues with getting that approach (`testthat::test_file`) to throw a proper error when a test fails.

I'd prefer to do it this way, and it's only [a handful of extra text files](https://github.com/JasjeetSekhon/rgenoud/tree/fbe954973680ea3dfffda1eba4d30109cb6f9487/tests/testthat) (< 10 KB).